README.md

```diff
@@ -94,6 +94,36 @@ Then when you use the `admin` profile, `aws-vault` will look in the `read-only`
 
 **Note:** If you have an MFA device attached to your account, the STS service will generate session tokens that are *invalid* unless you provide an MFA code. To enable MFA for a profile, specify the `mfa_serial` in `~/.aws/config`. You can retrieve the MFA's serial (ARN) in the web console, or you can usually derive it pretty easily using the format `arn:aws:iam::[account-id]:mfa/[your-iam-username]`. If you have an account with an MFA associated, but you don't provide the IAM, you are unable to call IAM services, even if you have the correct permissions to do so. `mfa_serial` will not be inherited from the profile designated in `source_profile` - you must include a reference to `mfa_serial` in every profile you wish to use it with.
 
+Alternatively, you can also specify your `mfa_serial` on your source profile. This can be very convenient if you routinely assume multiple roles from the same source because you will only need to provide an MFA token once per source profile session:
```

Comment:
The above line has a sentence which these changes contradict: _`mfa_serial` will not be inherited from the profile designated in `source_profile`_. From reviewing the changes you made, I think we can clean up the docs a bit. Removing the last sentence from L95, I would replace your docs with the following to clear up how the behavior has changed. Also fixed some of the source_profile examples to be working and the double mfa_serial=mfa_serial= in the first example.

---

`mfa_serial` will be inherited from the profile designated in `source_profile`, this can be very convenient if you routinely assume multiple roles from the same source as a MFA token is only provided once per `source_profile` session. You can always omit the `mfa_serial` on the `source_profile` to make MFA role-specific.

Here is an example where two roles defined in profile's `admin-a` & `admin-b` do not have the mfa_serial attribute, inheriting from the `master` profile instead.

```ini
[profile master]
mfa_serial = arn:aws:iam::123456789012:mfa/jonsmith

[profile admin-a]
source_profile = master
role_arn = arn:aws:iam::123456789012:role/admin-access

[profile admin-b]
source_profile = master
role_arn = arn:aws:iam::987654321987:role/admin-access
```
The developer only needs to configure their master profile; the list of profile config can be copied from a wiki (this is very useful when you have 25+ aws accounts with multiple roles in each).

You can also define a chain of roles to assume:
```ini
[profile master]
mfa_serial=mfa_serial = arn:aws:iam::123456789012:mfa/jonsmith

[profile intermediary]
source_profile = master
role_arn = arn:aws:iam::123456789012:role/intermediary

[profile target]
source_profile = intermediary
role_arn = arn:aws:iam::123456789012:role/target
```

Reply:
Ah, good suggestions. I incorporated your feedback in a new commit, but I didn't bother changing the profile name from `read-only` to `master` b/c I thought it was preferable to use names consistent with the other examples.